### DISCUSSION TITLE
Sensitive information API

### DISCUSSION THREAD

**Discussion by josueascencio** [Mimetis/Dotmim.Sync#546] [Q&A] [answered]
Dear,
How could I change the sensitive information displayed in the API view?

I am using DMS.Web.Server (0.8.0) and DMS.SqlServer.Change.Tracking (0.8.0)

Thank you very much

![image](https://user-images.githubusercontent.com/45615031/116508502-c2a6f780-a87e-11eb-9d0e-4e9d533e20f8.png)

**Comment by Mimetis** [comment, marked as answer]
When you will deploy in Production, the web page will be blank (with just a single message saying these information are hidden)

If you want to test it before, just set the environment variable `ASPNETCORE_ENVIRONMENT` from `Development` to `Production`

**Reply by josueascencio** [reply]
Thanks for your answer. 
I get a message that says: "Server is configured to Production mode. No options displayed."

There is no way to show the previous page but only to remove the sensitive information?
It is a good documentation for clients to know that the API is online.

**Reply by Mimetis** [reply]
Good idea. It's not implemented yet, but it could be, in a next release

**Comment by naugustijr** [comment]
Any news regarding this topic?
